modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA command line interface (CLI).
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+
+.Procedure
+
+* Create a machine pool with a custom tag by running the following command:
++
+--
+[source,terminal]
+----
+$ rosa create machinepools --cluster=<name> --replicas=<replica_count> \
+     --tags=<key>=<value>,<effect>,<key>=<value>,<effect> \ <1>
+----
+<1> Replace `<key>=<value>,<effect>,<key>=<value>,<effect>` with a key, value, and effect for each tag.
+--
++
+.Example output
+[source,terminal]
+----
+$ rosa create machinepools --cluster=mycluster --replicas 2 --tags=key1:NoSchedule,key2:NoExecute --name=mp-1
+
+I: Checking available instance types for machine pool 'mp-1'
+I: Machine pool 'mp-1' created successfully on cluster 'mycluster'
+I: To view the machine pool details, run 'rosa describe machinepool --cluster mycluster --machinepool mp-1'
+I: To view all machine pools, run 'rosa list machinepools --cluster mycluster'
+----
+//
+//. Add or update the tags for an existing machine pool:
+//.. Get the worker node ID of the machine pool you want to edit by running the following command:
+//+
+//[source,terminal]
+// //----
+// //$ rosa list machinepools --cluster=<cluster_name>
+// ----
+// +
+// .Example output
+// +
+// [source,terminal]
+// ----
+// ID      AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS    AVAILABILITY ZONES    SUBNETS    SPOT INSTANCES  DISK SIZE  SG IDs
+// mp-2    No           2         m5.xlarge                          us-east-1a                       No              300 GiB
+// worker  No           2         m5.xlarge                          us-east-1a                       No              300 GiB
+// ----
+
+// .. Add or update tags to an existing worker node in a machine pool by running the following command:
+// +
+// [source,terminal]
+// ----
+// $ rosa edit machinepool --cluster=<cluster_name> <machinepool_id> --tags=<key1:value1>,<key2:value2>
+// ----
+// +
+// .Example output
+// [source,terminal]
+// ----
+// $ rosa edit machinepool --cluster=mycluster mp-2 --tags=key1:NoSchedule,key2:NoExecute
+//
+// I: Updated machine pool 'mp-2' on cluster 'mycluster'
+// ----
+//
+.Verification
+
+. Use the `describe` command to see the details of the machine pool with the tags:
```

Comment:
@EricPonvelle here, you switched from bullet to number.  Not saying you should do one or the other, it's your call.